docs/src/operations/authentication.md

```diff
@@ -0,0 +1,172 @@
+# Authentication
+
+Zeebe supports transport layer security between the gateway and all of the officially supported clients. In this section, we will go through how to configure these components.
+
+## Gateway
+
+Transport layer security in the gateway is disabled by default. This means that if you're just experimenting with Zeebe or in development, there's no configuration needed. However, if you'd like to to enable authentication you can configure Zeebe in the `security` section of the configuration files. The following configurations are present in both `gateway.cfg.toml` and `zeebe.cfg.toml`, the file you should edit depends on whether you're using a standalone gateway or an embedded gateway.  
+
+```toml
+# Enables TLS authentication between clients and the gateway
+# This setting can also be overridden using the environment variable ZEEBE_GATEWAY_SECURITY_ENABLED. 
+# enabled = false
+#
+# Sets the path to the certificate chain file
+# This setting can also be overridden using the environment variable ZEEBE_GATEWAY_CERTIFICATE_PATH.
+# certificateChainPath = ""
+#
+# Sets the path to the private key file location
+# This setting can also be overridden using the environment variable ZEEBE_GATEWAY_PRIVATE_KEY_PATH. 
+# privateKeyPath = ""
+```
+
+The first setting simply enables the authentication feature, set it to `true` if you'd like to enable it. The second setting is a path that should point to a certificate chain in the PEM format. This should contain the gateway's certificate along a chain of any others necessary for the client to authenticate it. The third settings should be the path to the gateway's PKCS8 private key which should also be in the PEM format.
```

Comment:
Instead of first/second/third, I would propose to just use their names: 

`enabled` should be `true` or `false`, where true will enable TLS authentication between client and gateway, and false will disable it, ignoring any other settings. 
`certificateChainPath` and `privateKeyPath` are used to configure the certificate with which the server will authenticate itself. `certificateChainPath` should be a file path pointing to a certificate chain in PEM format representing the server's ceritificate, and `privateKeyPath` a file path pointing to the certificate's PKCS8 private key, also in PEM format.

Or something like this.
`privateKeyPath`

Reply:
> ignoring any other settings

I understand you mean the next two settings but wouldn't that make users think it'll ignore the corresponding environment variable?

Reply:
Hm, possibly. I guess we can cut that part.